server/views/account/update-email.jade

```diff
@@ -0,0 +1,57 @@
+extends ../layout
```

Comment:
We already have an `update-email` page [on beta](https://beta.freecodecamp.org/settings/update-email), Could this be used?

Reply:
That's what I tried first. If you try to hit `/settings` or `/settings/` pages from express redirect, the errors notifications are messed up. Instead of the flash, I get react notifications. And those are made generic 'Oops....' messages.

Also, I believe there are more than one for that matter.

I'll push a commit just to show what I mean.

Reply:
Are you getting any errors back?

Reply:
On the jade page, nope. I think the API endpoint was too customised for the react page. I have no clue what was the thought then. Anyways.. am okay with using the react page as long as we can figure out.. the reason.

Let me know what you think.. I am running out of ideas. I am around to pair.